peer-prep/src/matching-service/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:18
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
+
+EXPOSE 8080
```

Comment:
Yes, I see the importance of standardisation in configuring the external services we need to ease our project development! In addition, we also need to setup a uniform MongoDB host address and dependencies file to avoid conflicts.